src/goRunTestCodelens.ts

```diff
@@ -113,8 +113,28 @@ export class GoRunTestCodeLensProvider extends GoBaseCodeLensProvider {
 				};
 
 				codelens.push(new CodeLens(func.location.range, runBenchmarkCmd));
-			});
 
+				const args = [
+					'-test.bench', func.name,
+					'-test.run', 'a^' // Don't match any test
+				]; 
+				const program = path.dirname(document.fileName);
```

Comment:
A lot of code here is similar to whats needed for the debug test codelens. Instead of repeating it, it would be better to pull the common code out of the promise creation